packages/mobile/src/components/core/Button.tsx

```diff
@@ -243,6 +245,16 @@ export const Button = (props: ButtonProps) => {
     [onPressOut, handlePressOutScale, handlePressOutColor]
   )
 
+  /*
+   * Ensures button take up a static height even when scaling
+   */
+  const handleRootLayout = useCallback(
```

Comment:
Good catch, I wonder if this could/should be incorporated in the `usePressScaleAnimation` hook?

Reply:
yeah i wonder as well, how would that hook figure out the height of the root element i guess?